scripts/migration.sh

```diff
@@ -164,5 +177,5 @@ done
 # Push git repository
 if [ "${push_repository}" = true ]
 then
-    git -C ${l10n_path} push
+    git -C ${l10n_path} push 2>/dev/null || git -C ${l10n_path} push --set-upstream origin ${branch_name}
```

Comment:
```suggestion    
    if [ "${create_branch}" = true ]
    then
        git -C ${l10n_path} push --set-upstream origin ${branch_name}
    else
        git -C ${l10n_path} push
    fi
```

Would this be clearer?

Reply:
Also, probably safer, since an unrelated failure in `git push` would try to create the branch.

Reply:
Yeah that makes sense. Will reflect now.